src/L2GovernorMetadata.sol

```diff
@@ -63,4 +89,15 @@ abstract contract L2GovernorMetadata {
   function getProposal(uint256 proposalId) public view virtual returns (Proposal memory) {
     return _proposals[proposalId];
   }
+
+  /// @notice Calculate the approximate block that the L2 will be producing at the time the
+  /// L1 produces some given future block number.abi
```

Comment:
```suggestion
  /// L1 produces some given future block number.
```

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610